org-code-javabuilder/lib/src/main/java/org/code/neighborhood/GridFactory.java

```diff
@@ -75,4 +75,16 @@ protected Grid createGridFromString(String description) throws IOException {
       throw new IOException(ExceptionKeys.INVALID_GRID.toString());
     }
   }
+
+  // Creates an empty size x size grid with every square being open
+  // and having assetId 0.
+  protected Grid createEmptyGrid(int size) {
```

Comment:
This should also be public since it will be accessed from an outside package. Also, mind adding a quick note to the comment describing how someone should set up their code to use this? i.e. I think it's something like `new World(); world.getFactory().createEmptyGrid(8)` right?

Reply:
Oops. I take that back.